components/eamxx/cime_config/namelist_defaults_eamxx.xml

```diff
@@ -193,8 +193,10 @@ be lost if SCREAM_HACK_XML is not enabled.
     <p3 inherit="atm_proc_base">
       <do_prescribed_ccn>true</do_prescribed_ccn>
       <use_hetfrz_classnuc type="logical" doc="Switch to turn on heterogeneous freezing due to prognostic aerosols">false</use_hetfrz_classnuc>
```

Comment:
Is this feature (CNT ice nucleation) considered immature to be turned on for MAM4xx?

Reply:
Yes, that's what the plan is. I will do that in a separate PR.

Reply:
> Is there any other flag we need to change?

No, not I am aware of.